src/main/java/draughts/Board.java

```diff
@@ -125,48 +125,48 @@ public void movePawn(Pawn pawn, Coordinates position){
         //There is a movePawn() method that moves pawns from a specified position to another field.
         //This method is just changing pawns coordinates
         //removes pawn from startPosition and moves it to endPosition
-        int startX = pawn.getPosition().getX();
-        int startY = pawn.getPosition().getY();
-        this.fields[startX][startY] = null;
-        int goalX = position.getX();
-        int goalY = position.getY();
+        int startRow = pawn.getPosition().getRow(); // startX -> startRow
+        int startCol = pawn.getPosition().getCol();// startY -> startCol
+        this.fields[startRow][startCol] = null;
+        int goalRow = position.getRow(); // goalX -> goalRow
+        int goalCol = position.getCol(); // goalY -> goalCol
         pawn.setPosition(position);
-        this.fields[goalX][goalY] = pawn;
+        this.fields[goalRow][goalCol] = pawn;
     }
 
     public void createBoard(){
         // one side of the board
-        int n = this.getBoardSize();
-        int numberOfPawns = n * 2;
+        int sideLength = this.getBoardSize();
+        int numberOfPawns = sideLength * 2;
 
-        for (int i = 0; i < n; i++) {
-            for (int j = 0; j < n; j = j + 2) {
-                if (i % 2 == 0) {
+        for (int rows = 0; rows < sideLength; rows++) {
+            for (int columns = 0; columns < sideLength; columns = columns + 2) {
```

Comment:
Sometimes variable is named `col(s)`, sometimes `column(s)`. Consider using the same format